bin/subiquity-service

```diff
@@ -5,7 +5,13 @@ export PYTHONPATH=$SNAP/lib/python3.8/site-packages
 if [ -n "$1" ]; then
    port=$1
 fi
+
+# stops some dmesg from overwriting the TUI
 /bin/dmesg -n 1
+
+# stop the systemd service completion messages from doing the same
+kill "-SIGRTMIN+21" 1
```

Comment:
hnnngh i don't think dash's kill builtin supports this syntax (hilariously, dash's kill builtin wasn't documented at all until a few weeks ago). I'm also not sure this syntax is right for dash even if it was supported. Anyway, try changing it to `/bin/kill -SIGRTMIN+21 1` ?

Reply:
yep.. dash's kill does not support the `kill -SIGINT` syntax. Only `kill -INT` or `kill -s INT`

I tried it on my system and it seems like it can't go beyond RTMIN+15.
```
$ dash -c 'kill -s RTMIN+14 99999'
dash: 1: kill: No such process
$ dash -c 'kill -s RTMIN+15 99999'
dash: 1: kill: No such process
$ dash -c 'kill -s RTMIN+16 99999'
dash: 1: kill: invalid signal number or name: RTMIN+16
```
```
$ dash -c 'kill -l' | grep RTMIN
RTMIN
RTMIN+1
[...]
RTMIN+14
RTMIN+15
```
> Anyway, try changing it to /bin/kill -SIGRTMIN+21 1 ?

+1

Reply:
I'm proposing `/bin/kill "-55" 1`, as in dash in this environment, neither `kill -l` nor `/bin/kill -l` are giving me high confidence.  But they both seem to work with the number just fine.

Reply:
fwiw man 7 signal is pretty strict regarding this rule but I don't know if we're really at risk of doing something wrong... so I'd say pick what you think it best.

>  programs should  never refer to real-time signals using hard-coded numbers, but instead should always refer to real-time signals using the notation SIGRTMIN+n

[1] https://www.man7.org/linux/man-pages/man7/signal.7.html#DESCRIPTION § Real-time signals

Reply:
gross.  I'm going to read the source for /bin/kill and do whatever that suggests.

Reply:
Sorry, why don't you like "/bin/kill -SIGRTMIN+21 1" ? I agree it doesn't appear in the -l output but I don't think it's going to stop working. I guess you could do something like kill $(($(/bin/kill -l SIGRTMIN)+21)...

A NACK on hardcoding 55 here in any case.

Reply:
eh that doesn't work either. maybe we should just use bash for this helper :-)

Reply:
Yes, because it doesn't match what /bin/kill shows as expected.  If the goal here is to maximize forward compatibility then my hope is that by conforming to what /bin/kill wants, we can do that.

Reply:
https://git.launchpad.net/ubuntu/+source/procps/tree/proc/sig.c#n139 suggests that support for a construction like "-SIGRTMIN+21" is reasonable to assume.